T_marker_cam: 
  - [0.04132279, -0.99908152, 0.01133729, -0.04988]
  - [-0.34024411, -0.02473974, -0.94001164, -0.07559]
  - [0.93942875, 0.03498646, -0.34095392, -0.12605]
  - [0.00000000, 0.00000000, 0.00000000, 1.00000000]


